wsd/DocumentBroker.cpp

```diff
@@ -313,6 +313,12 @@ void DocumentBroker::pollThread()
         if (_tileCache)
             _tileCache->setMaxCacheSize(8 * 1024 * 128 * _sessions.size());
 
+        if (isInteractive())
+        {
+            loadDeadline = now + std::chrono::seconds(limit_load_secs);
+            continue;
```

Comment:
How do we get out of interactive mode?

I expect we want to set interactive to false at some point so that we don't always `continue` here and execute the remaining conditions in this loop.